lib/xcpretty/formatters/formatter.rb

```diff
@@ -176,18 +177,18 @@ def format_failures(failures_per_suite)
       end.join("\n")
     end
 
-    def format_failure(f)
-      snippet = Snippet.from_filepath(f[:file_path])
-      output = "  #{f[:test_case]}, #{red(f[:reason])}"
-      output += "\n  #{cyan(f[:file_path])}"
+    def format_failure(failure)
+      snippet = Snippet.from_filepath(failure[:file_path])
+      output = "  #{failure[:test_case]}, #{red(failure[:reason])}"
+      output += "\n  #{cyan(failure[:file_path])}"
       return output if snippet.contents.empty?
 
       output += "\n  ```\n"
-      if @colorize
-        output += Syntax.highlight(snippet)
-      else
-        output += snippet.contents
-      end
+      output += if @colorize
```

Comment:
Maybe use `?:` operator to use less vertical space.

Reply:
this works